src/commands/CodingJamsCommand.ts

```diff
@@ -255,6 +257,8 @@ const pollSelectionStringOption = new SlashCommandStringOption()
 		{ name: "Random", value: pollSelectionTypes.random },
 		{ name: "Newest", value: pollSelectionTypes.newest },
 		{ name: "Oldest", value: pollSelectionTypes.oldest },
+		{ name: "Fewest polls", value: pollSelectionTypes.fewestPolls },
+		{ name: "Most polls)", value: pollSelectionTypes.mostPolls },
```

Comment:
Looks like there is a imposter (the closed bracket)

Reply:
yes qwq